test/functional/page_objects/settings_page.js

```diff
@@ -154,44 +144,39 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
       return await testSubjects.find('indexPatternTitle');
     }
 
-    getConfigureHeader() {
-      return remote.setFindTimeout(defaultFindTimeout)
-        .findByCssSelector('h1');
+    async getConfigureHeader() {
+      return await find.allByCssSelector('h1');
```

Comment:
Should be `find.byCssSelector('h1')` instead of `find.allByCssSelector`.  The "all*" variations actually take a lot longer than the singular ones, like seconds longer.